importDBdmpFile/check_db_dump_recent.py

```diff
@@ -47,26 +51,29 @@
     # "Couldn't open/read the provided file" differently.
     try:
         with open(log_file, 'r') as log_file:
-            lines = log_file.read().splitlines()
-            last_line = lines[-1]
+            line_list = log_file.read().splitlines()
```

Comment:
Bit of a concern about performance as you're extracting the whole file. Does the log file get rotated regularly enough for this not to be an issue?

(There are ways, if you open a file in binary mode, to seek from the end.)

Reply:
I've left this as is in #21. I think it's more readable than something like `seek()` and it extracting the whole file is not currently a performance issue.